warp10/src/main/java/io/warp10/script/mapper/MapperParseDouble.java

```diff
@@ -0,0 +1,112 @@
+package io.warp10.script.mapper;
+  
+  //
+//Copyright 2016  Cityzen Data
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+//
+
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptMapperFunction;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+import io.warp10.script.WarpScriptException;
+
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+* Mapper which returns the absolute value of the value passed as parameter
+*/
+public class MapperParseDouble extends NamedWarpScriptFunction implements WarpScriptMapperFunction {
+
+private final NumberFormat format;
+  
+  public MapperParseDouble(String name) {
+    super(name);
+    this.format = null;
+  }
+  
+  public MapperParseDouble(String name, Object language) throws WarpScriptException {   
+    super(name);
+    
+    if (language instanceof String) {
+      Locale locale = Locale.forLanguageTag((String) language);
+      format = NumberFormat.getInstance(locale);
+    } else {
+      throw new WarpScriptException("Invalid value type for " + getName() + ", expects a String");
```

Comment:
Change message to (getName() + " expects a language tag (a string) on top of the stack.")